ui/Makefile

```diff
@@ -42,8 +44,8 @@ debug: $(GOBINDATA_DEBUG_TARGET)
 	rm -f $(GOBINDATA_TARGET)
 
 $(TYPESCRIPT_TARGET): $(shell find ts)
```

Comment:
this should be `$(shell find $(TS_ROOT))` now that you've changed all the args below